.travis.yml

```diff
@@ -96,7 +96,7 @@ install:
   - source ~/.install-jdk-travis.sh
 
 script:
-  - if [ "$API" != "NONE" ]; then ./gradlew test check connectedCheck -x lint --stacktrace; fi
+  - if [ "$API" != "NONE" ]; then ./gradlew test check jacocoTestReport connectedCheck -x lint --stacktrace; fi
```

Comment:
`test` executes the JUnit tests we have in the `library/src/test`, and `connectedCheck` executes the instrumented tests we have in `library/src/androidTest`. Do you know if `jacocoTestReport` covers the output of both of these?

Reply:
`jacocoTestReport` will only evaluate the existence of a test (unit or instrumented test) for a particular set of classes (i.e. classes within the `:library` module). The actual result of the unit and instrumented tests are currently handled by Travis.

Example CodeCov result:
![image](https://user-images.githubusercontent.com/463186/74859906-aa436080-52fc-11ea-8cd8-2b167d749f97.png)